.pre-commit-config.yaml

```diff
@@ -1,7 +1,9 @@
 repos:
--   repo: https://github.com/dotnet/format
-    rev: v5.1.225507
+-   repo: local
     hooks:
     -   id: dotnet-format
-        entry: dotnet-format Source/Dafny.sln -w --
-        
\ No newline at end of file
+        name: dotnet-format
+        language: system
+        entry: bash -c 'cd Source && dotnet tool run dotnet-format Dafny.sln -w --'
```

Comment:
You mention this probably doesn't work on windows, but we do support windows development. Would you mind putting this in a separate PR ?

Reply:
Removed this commit from this PR